### ISSUE TITLE
False positive in ext.js Format module

### ISSUE THREAD

**Issue by dnet** [augustd/burp-suite-error-message-checks#45]
Source: https://github.com/pjreddie/WRed/blob/3b446e19643b1e298933b622ba72929f362713f2/media/JavaScript/ext-3.2.1/src/util/Format.js#L296

```js
throw ('NumberFormatException: invalid format, formats should have no more than 1 period: ' + format);
```

The above line matches the regex `([A-Za-z\.]+)Exception:` which results in a false positive.

**Comment by augustd** [comment]
This is an example ASP.Net error message we are trying to match against: 
```
System.Web.Services.Protocols.SoapException: Web Exception ---&gt; System.Exception:
```
It will be difficult to rule out this false positive without also creating a false negative as well.

**Comment by dnet** [comment]
I think you fixed this in e72368de (#49). If you agree, you can close this one.
